viite-UI/src/view/RoadAddressProjectForm.js

```diff
@@ -157,16 +157,20 @@
       };
 
       eventbus.on('roadAddress:newProject', function() {
+        currentProject=undefined; //clears old data
+        $("#roadAddressProject").html("");
         rootElement.html(newProjectTemplate());
         jQuery('.modal-overlay').remove();
         addDatePicker();
         applicationModel.setOpenProject(true);
+        projectCollection.clearRoadAddressProjects();
       });
 
       eventbus.on('roadAddress:openProject', function(result) {
         currentProject = result.projects;
+        projectCollection.clearRoadAddressProjects();
         var text = '';
-        _.each(result.projectLinks, function(line){
+        _.each(result.projectLinks, function(line){  //TODO later list of already saved roadlinks has to be saved in  roadaddressprojectcollection.currentRoadSegmentList for reserve button to function properly now saved links are cleared when newones are reserved
```

Comment:
Sorry to bother @oappicgi, but is this TODO, to be done in a separate task?

I ask this since it was this [commit](https://github.com/finnishtransportagency/digiroad2/pull/694/commits/988477a407f837635028675bdf3d851669c14a9e) that introduced the TODO.